tests/cnf/core/network/sriov/tests/metricsExporter.go

```diff
@@ -397,44 +403,56 @@ func defineDPDKPod(role, devType, worker string) *pod.Builder {
 	return dpdkPod
 }
 
-func createTestResources(testRes testResource) *pod.Builder {
-	By(fmt.Sprintf("Creating SriovNetworkNodePolicy %s", testRes.policy.Definition.Name))
+func createTestResources(cRes, sRes testResource) (*pod.Builder, *pod.Builder) {
+	By("Create SriovNetworkNodePolicy")
+
+	for _, res := range []testResource{cRes, sRes} {
+		_, err := res.policy.Create()
+		Expect(err).ToNot(HaveOccurred(),
+			fmt.Sprintf("Failed to Create SriovNetworkNodePolicy %s", res.policy.Definition.Name))
+	}
 
-	err := sriovenv.CreateSriovPolicyAndWaitUntilItsApplied(testRes.policy, tsparams.MCOWaitTimeout)
-	Expect(err).ToNot(HaveOccurred(), "Failed to create SriovNetworkNodePolicy")
+	err := netenv.WaitForSriovAndMCPStable(APIClient, tsparams.MCOWaitTimeout, tsparams.DefaultStableDuration,
+		NetConfig.CnfMcpLabel, NetConfig.SriovOperatorNamespace)
 
-	By(fmt.Sprintf("Creating SriovNetwork %s", testRes.network.Definition.Name))
+	By("Create SriovNetwork and wait for NAD creation")
 
-	_, err = testRes.network.Create()
-	Expect(err).ToNot(HaveOccurred(), "Failed to create SR-IOV network")
+	for _, res := range []testResource{cRes, sRes} {
```

Comment:
Just suggestion:
You can create policy and network in the same itteration. And then wait for MCP stable and nad verify that nad has been created

Reply:
done